incubator/validator-core/src/main/java/io/aklivity/zilla/runtime/validator/core/StringValidatorFactory.java

```diff
@@ -39,11 +40,38 @@ public URL schema()
     }
 
     @Override
-    public Validator create(
+    public ValueValidator createValueReader(
```

Comment:
Now that validator is created by engine, can engine automatically assign `id` on `ValidatorConfig` instead of passing `resolveId` around?